migration/migrating_3_4/deploying-cam-3-4.adoc

```diff
@@ -1,68 +1,68 @@
 [id='deploying-cam-3-4']
-= Deploying the Migration Toolkit for Containers
+= Installing and upgrading the Migration Toolkit for Containers
 include::modules/common-attributes.adoc[]
 :context: migrating-3-4
 :migrating-3-4:
 
 toc::[]
 
-You can deploy the {mtc-full} ({mtc-short}) on an {product-title} {product-version} target cluster and an {product-title} 3 source cluster by installing the {mtc-full} Operator.
+You can install the {mtc-full} ({mtc-short}) on an {product-title} {product-version} target cluster and an {product-title} 3 source cluster.
 
-[NOTE]
-====
-{mtc-short} is installed on the target cluster by default.
-
-You can configure the {mtc-full} Operator to install the {mtc-short} link:https://access.redhat.com/articles/5064151[on an {product-title} 3 cluster or on a remote cluster].
-====
-
-In a restricted environment, you can install the {mtc-full} Operator from a local mirror registry.
-
-After you have installed the {mtc-full} Operator on your clusters, you can launch the {mtc-short} console.
+{mtc-short} is installed on the target cluster by default. You can install {mtc-short} link:https://access.redhat.com/articles/5064151[on an {product-title} 3 cluster or on a remote cluster].
 
-[id='installing-cam-operator_{context}']
-== Installing the {mtc-full} Operator
+[id='installing-mtc_{context}']
+== Installing the {mtc-full} in a connected environment
 
-You can install the {mtc-full} Operator with the Operator Lifecycle Manager (OLM) on an {product-title} {product-version} target cluster and manually on an {product-title} 3 source cluster.
+You can install the {mtc-full} ({mtc-short}) in a connected environment.
 
 [IMPORTANT]
 ====
-You must ensure that the same Operator version is installed on the source and target clusters.
+You must install the same {mtc-short} version on all clusters.
 ====
 
-include::modules/migration-installing-cam-operator-ocp-4.adoc[leveloffset=+2]
-include::modules/migration-installing-cam-operator-ocp-3.adoc[leveloffset=+2]
+include::modules/migration-installing-mtc-on-ocp-4.adoc[leveloffset=+2]
+include::modules/migration-installing-mtc-on-ocp-3.adoc[leveloffset=+2]
 :!migrating-3-4:
 
 :context: disconnected-3-4
 :disconnected-3-4:
-[id='installing-cam-operator_{context}']
-== Installing the {mtc-full} Operator in a restricted environment
+[id='installing-mtc_{context}']
+== Installing the {mtc-full} in a restricted environment
 
-You can install the {mtc-full} Operator with the Operator Lifecycle Manager (OLM) on an {product-title} {product-version} target cluster and manually on an {product-title} 3 source cluster.
+[IMPORTANT]
```

Comment:
Do you want a _You can install the Migration Toolkit for Containers_ sentence here, like in the other sections?